packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -64,7 +68,10 @@ export const PlanDetails = ({
         >
           <div className="plan-details-header">
             <div className="plan-details-header-wrap">
-              <div className="plan-details-logo-wrap">
+              <div
+                className="plan-details-logo-wrap"
+                style={{ ...setWebIconBackground }}
```

Comment:
I guess if the markup around the icon ever gets more complex, we should consider extracting it out to a component that does stuff like the `setWebIconBackground` logic. But not worth holding this PR up for